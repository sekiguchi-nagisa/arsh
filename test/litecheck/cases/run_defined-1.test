# RUN: /bin/sh $self
# RUN: echo hello
# CHECK: hello