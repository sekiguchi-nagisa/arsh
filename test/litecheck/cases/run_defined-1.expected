run_defined-1.test:2: [error] RUN directive has already defined